pkg/autodiscover/autodiscover_pv.go

```diff
@@ -39,3 +42,17 @@ func getPersistentVolumeClaims(oc corev1client.CoreV1Interface) ([]corev1.Persis
 	}
 	return pvcs.Items, nil
 }
+
+func getAllStorageData() (data []string, err error) {
```

Comment:
`err` is not handled in the calling func.

Reply:
you're right! I fixed it.